server.js

```diff
@@ -39,14 +43,31 @@ app.get("/menu_items", function (req, res) {
   ]);
 });
 
-//Post Method
-app.post("/menu_items", (req, res, next) => {
-  res.send(JSON.stringify(req.body));
+app.post("/food", async (req, res) => {
+  const food = await prisma.food.create({ data: req.body });
+  res.json(food);
 });
 
-//Delete by ID Method
-app.delete("/delete/:id", (req, res) => {
-  res.send("Delete by ID API");
+//bad request
+app.delete("/food/:id", async (req, res) => {
+  const { id } = req.params;
+  const deletedFood = await prisma.food.delete({
+    where: {
+      id: Number(id),
+    },
+  });
+  res.json(deletedFood);
+});
+
+app.put("/food/:id", async (req, res) => {
+  const { id, img } = req.params;
```

Comment:
I'd love to see `img` come from `req.body` instead of the params.  That would be more consistent with PUT semantics where it's common to only see the `id` in the URL and everything else in the body.

Also, a `PUT` ultimately means "replace the entire record at the provided ID with the payload in the `body`.  As written, this code will not update the food item's title or author, which I _assume_ it should unless you're considering those to be protected fields that can't be updated?

Reply:
Thank you - Making the updates